docs/install.rst

```diff
@@ -9,292 +9,60 @@ Simple Installation
-
-.. note:: 
-    There are two main authentication schemes for GitHub: SSH and HTTPS. The examples below use URLs containing ``git@github.com``, which assumed that you are usingg SSH authentication. If you have not already enabled ssh keys on your GitHub account, you can follow the `GitHub instructions for setting up ssh keys <https://help.github.com/articles/generating-ssh-keys/>`_ to set this up. Alternatively, you can use the HTTPS connection method, where ``git@github.com:`` is replaced with ``https://github.com/``. See the `GitHub documentation on URLs <https://help.github.com/articles/which-remote-url-should-i-use/>`_ for more information. You can also read the `pip git instructions <https://pip.pypa.io/en/latest/reference/pip_install.html#git>`_ for more details on how to install a branch or a specific tag.
-
-Install the PyCBC source code from the GitHub URL using the command:
+We can then make a fresh clone of the repository.
 
 .. code-block:: bash
 
-    pip install -e git+git@github.com:your-username-here/pycbc.git#egg=pycbc
+    git clone git@github.com/gwastro/pycbc.git
 
-This will fetch the PyCBC source and will also install all the listed dependencies. The ``-e`` option to pip creates a directory called ``${VIRTUAL_ENV}/src/pycbc`` with a git checkout which is fully editable. To prevent pip from removing this source directory run the command
-
-.. code-block:: bash
-
-    rm -f ${VIRTUAL_ENV}/src/pip-delete-this-directory.txt
-
-You can then make changes to your PyCBC source code in the directory ``${VIRTUAL_ENV}/src/pycbc``
-
-You can also use the repository created by pip as your working repository, creating branches, commits, and `pull requests <https://help.github.com/articles/creating-a-pull-request/>`_ as you need to. To keep your repository in sync with the ligo-cbc/pycbc repository, you can read the GitHub instructions that explain how to `sync a fork of a repository to keep it up-to-date with the upstream repository. <https://help.github.com/articles/syncing-a-fork/>`_.
-
-.. note::
-
-    The version of PyCBC that is checked out will be on the master branch. To track a remote branch from your GitHub repository, run the command
-    
-    ``git branch --set-upstream branch_name origin/branch_name``
-
-    ``git checkout branch_name``
-        
-    where branch_name is the name of the branch that you want to track.
-
-To build and install any changes that you make to the source code in your virtual environment, run the command
+Finally, install the most common pycbc develeopment environment packages
+as follows.
 
 .. code-block:: bash
 
+    cd pycbc
+    pip install -r requirements.txt
```

Comment:
Can we also highly recommend users to install `companion.txt` as well, unless they need the diskspace.

Reply:
Ok, added that in.